### DISCUSSION TITLE
Startups creator

### DISCUSSION THREAD

**Discussion by RS-RICH-DE-ANGOLA-12** [community/community#137457] [Programming Help]
### Body

I'm creating a startup that I believe will change the education and technology industry, I'm looking for partners to also own the startups, you have to be good at programming and also good at integrating AI

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by thirumaleshp** [comment]
Interesting...

**Comment by AkshayMoolya** [comment]
Great! Let's connect to discuss your startup idea in more detail.

**Comment by Azymack** [comment]
Hi, mate.
You can connect me on Linkedin or email.
https://www.linkedin.com/in/stefan-jeremic-147bb0322/
sleekj13@gmail.com
